tests/flow/test_graph_deletion.py

```diff
@@ -164,6 +164,8 @@ def test09_delete_entire_graph(self):
         # Try to query a deleted graph should raise an exception.
         try:
             redis_graph.query(query)
-            self.env.assertTrue(False)
+            result = redis_graph.query(query)
+            nodeCount = result.result_set[0][0]
+            self.env.assertEquals(nodeCount, 0)
         except:
-            pass
+            self.env.false()
```

Comment:
`self.env.false()` is the same as `self.env.assertTrue(False)` ?

Reply:
This change was my suggestion, the code is:
https://github.com/RedisLabsModules/RLTest/blob/master/RLTest/env.py#L72-L74

I feel like it more cleanly resembles methods like GoogleTest's `FAIL()`.